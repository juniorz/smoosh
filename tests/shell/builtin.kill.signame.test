rm -f foo
set -e

trap 'touch foo' TERM

kill $$
[ -f foo ] && ! [ -s foo ]
rm foo
echo plain kill

kill -TERM $$
[ -f foo ] && ! [ -s foo ]
rm foo
echo named \(-TERM\)

kill -15 $$
[ -f foo ] && ! [ -s foo ]
rm foo
echo numbered \(-15\)
