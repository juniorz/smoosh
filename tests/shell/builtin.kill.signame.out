plain kill
named (-TERM)
numbered (-15)
